src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3081,8 +3081,10 @@ def construct_mc_profile_preview(self, bypass_restore_defaults: bool = False) ->
         mc = self.set_up_image_integrity(mc)
         # set up cluster snapshot
         mc = self.set_up_creationdata_of_cluster_snapshot(mc)
-        # set up app routing profile
+        # set up app routing profile using `aks approuting enable`
         mc = self.set_up_app_routing_profile(mc)
+        #set up app routing profile using `aks enable-addons`
```

Comment:
```suggestion
        #set up app routing profile using `aks create --enable-addons web_application_routing`
```
the implementation of command `aks enable-addons` is in `custom.py`